dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -296,6 +438,83 @@ NavigationViewItemPresenter * NavigationViewItem::GetPresenter()
     return presenter;
 }
 
+void NavigationViewItem::IsRepeaterVisible(bool visible)
```

Comment:
the naming is a bit confusing since it looks like a query to find out if repeater is visible, but what we are doing has a bunch of side effects. Should it be SetRepeaterVisibility(..) ?